python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -112,17 +112,17 @@ def test_hypothesis_max_min_agg(lmdb_version_store, df):
     assert_frame_equal(expected, vit.data)
 
 
-def test_sum_aggregation(s3_version_store):
+def test_sum_aggregation(object_version_store):
```

Comment:
Most of the unit tests are not storage-dependent, so we should not just blindly replace all of them with `object_version_store`, making them take twice as long with limited additional bug detection power.